arch/ecmwf/hpc2020/gnu/11.2.0/env.sh

```diff
@@ -35,7 +35,7 @@ module_load hpcx-openmpi/2.10.0
 module_load boost/1.71.0
 module_load hdf5/1.10.6
 module_load cmake/3.20.2
-module_load python3/3.8.8-01
+module_load python3/3.10.10-01
```

Comment:
Sticking to a single version of Python on a given machine was a real pain to me through the process, this was just to make things a bit simpler. 3.10 becomes standard on ATOS in May, anyway. If I recall correctly, 3.6 was not cooperating, 3.8 should be OK.

Reply:
True, but if it's not strictly required for this PR, this change should be filed, discussed and tested separately, to avoid knock-on effects with other Python variants.